.github/workflows/lte-integ-test-bazel-magma-deb.yml

```diff
@@ -123,13 +123,11 @@ jobs:
           path: lte/gateway/logs.tar.gz
 
       - name: Notify failure to slack
-        if: failure() && github.event_name == 'push' && github.repository_owner == 'magma'
-        uses: rtCamp/action-slack-notify@12e36fc18b0689399306c2e0b3e0f2978b7f1ee7 # pin@v2.2.0
+        if: failure() && github.repository_owner == 'magma'
+        uses: Ilshidur/action-slack@689ad44a9c9092315abd286d0e3a9a74d31ab78a # pin@2.1.0
```

Comment:
`Ilshidur/action-slack` seems to be way less maintained and less popular as `rtCamp/action-slack-notify`

Reply:
used by 600 compared to used by 8k

Reply:
Yes, but rtCamps action does not work on macos runners.

Reply:
OK, was just surprised and thought it would be wroth asking,